panther_ioc_rules/sunburst_fqdn_iocs.py

```diff
@@ -0,0 +1,15 @@
+from panther_iocs import sunburst_fqdn_ioc_match, intersection, SUNBURST_FQDN_IOCS
```

Comment:
Should we have separate rules for IoC matches? Or a single rule that operates against multiple lists of IoCs?

Reply:
Good question, I could go either way on it to honest. I do kind of like separate rules as it gives you a little bit more knobs to turn with regards to which log types apply which logic, but it's not a big deal. I'm pushing up a change with three rules, but if you guys decide to reduce it to one it should be pretty easy.

Reply:
> Should we have separate rules for IoC matches? Or a single rule that operates against multiple lists of IoCs?

Single rule that operates against multiple lists of IoCs IMO

We can get that same benefit @nhakmiller by making a rule for each indicator type that we match on i.e. fqdn, ip, sha256